recipes/mrc/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "mrc" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mrc-{{ version }}.tar.gz
+  sha256: caab4c6cd91af4c0311256d97ffb3017ae9818002ce2edf9adc246510e3e49e0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - setuptools-scm
+  run:
+    - numpy
+    - python >=3.7
+
+test:
+  imports:
+    - mrc
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/tlambert03/mrc
+  summary: Read and write .mrc and .dv (deltavision) image file format
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - conda-forge/napari
```

Comment:
This one does not depend on napari, but I assume is a dependency of a plugin. Do we still want the team to be maintaining this one too?

Reply:
No not On this one. That would just be napari-dv

Reply:
Just me here